File: AirportManagement.API/AirportManagement.API/Data/Models/NoFlyList.cs
using System;
namespace AirportManagement.API.Models
{
	public class NoFlyList
	{
		public int Id { get; set; }

		public DateOnly ActiveFrom { get; set; }

		public DateOnly ActiveTo { get; set; }

		public string NoFlyReason { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public int PassangerId { get; set; }

		public Passangers Passangers { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/NoFlyList.ts
import { Passangers } from "./Passangers";

export interface NoFlyList {
    Id: number;
    ActiveFrom: Date;
    ActiveDo: Date;
    NoFlyReason: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    PassangerId: number;
    Passangers: Passangers;
}

export interface NoFlyListForPassanger {
        activeFrom: Date;
        activeTo: Date;
        noFlyReason: string,
        createdAt: Date,
        updatedAt: Date
}
